tutorials/io/60_ctf_bst_auditory.py

```diff
@@ -90,6 +90,9 @@
 #   - 12 head tracking channels
 #   - 20 unused channels
 #
+# Notice also that the digitized electrode positions (stored in a .pos file)
+# were automatically loaded and added to the `mne.io.Raw` object.
```

Comment:
```suggestion
# were automatically loaded and added to the `~mne.io.Raw` object.
```

Reply:
@rylaw FYI, the tilde will cause Sphinx to suppress the `mne.io.` part and will just display `Raw` in the rendered HTML (while still cross-linking to the appropriate API page).